### DISCUSSION TITLE
Synaptic Dark Mode

### DISCUSSION THREAD

**Discussion by Visikde** [SpiralLinux/SpiralLinux-project#464] [Q&A]
I have used various distros for some years as a simple user  
The occasions I've had to use CLI in 15 years have been rare  
I've been using Manjaro for a few years, I switched to Mageia [long time distro] to avoid the switch to Plasma 6  
Bought a 2tb nvme & more ram, time to start fresh  
My usual criteria:  
User friendly 
Stable 
Community built, no corporate, no one man shows  
Large enough community so there is some support  
A distro from each of the main forks  
Mageia does rpm [mageia control center, drak tools]
Manjaro arch [pamac is great]
Debian I never quite found what I was looking for 
MX is wild n woolly, been ok once they got over their init fixation  
Spiral, which appears to violate the no one man show rule, but is actually the biggest community of them all Debian stable! Flatpak if I'm looking for something newer, anything out in the wild that has any linux has a .deb, need a solution multiple communities
I installed Spiral [KDE] on an external drive a few weeks ago, Couldn't find a deal breaker :D   
I just finished installing Spiral on my new hardware & am working on recycling my old stuff to a friend, who just needs to do normal internet, photos, music, occasionally print a page kind of things, [mostly uses her phone]
Usually I would do Mageia KDE, but I think Spiral will be even easier, as deb help is easier to look up   

Enough personal boiler plate...  

I generally avoid having to use synaptic, as it's hard for me to read the tan print on the off white background  
Switching my monitor [system settings] to night & scaled up some helps some
Is there a gui way to make it dark mode?
 
Being on KDE it seems like I won't need to use Synaptic very much as Discovery may do enough of the package management stuff  

Do I need to add a root user?

**Comment by geckolinux** [comment]
Hi there, glad you're liking SpiralLinux.

> tan print on the off white background

Hmm, it should be black text by default.

At any rate, to change the theming of apps that run as root in Plasma / KDE you would need to change the KDE appearance settings for root by launching the settings app as root: `sudo systemsettings5` and then modifying **Appearance > Application Style** 

![Screenshot from 2024-08-21 23-02-10](https://github.com/user-attachments/assets/ef90c560-afab-41d5-8d7a-e152ef76054e)

**Reply by geckolinux** [reply]
> At any rate, to change the theming of apps that run as root in Plasma / KDE you would need to change the KDE appearance settings for root by launching the settings app as root:

Hi again, a bit of a clarification on this previous point, changing the appearance settings as root applies only to KDE / Qt apps that run as root. Since Synaptic is a GTK app you would need to change its style with the **Appearance > Application Style > [Configure GNOME/GTK Application Style...]** button, which does not appear when running KDE Settings as root. But changing the GTK theme in KDE Settings as a normal user also sets the same theme for GTK apps running as root, so that's what you would want to change in your case.

**Comment by Visikde** [comment]
Thanks for the reply 
I'm finding Discover able to do what I need for the most part, I keep checking with Synaptic, but Discover is better than the last time I used it
I'm happier having global scaled up my monitor some, since Synaptic doesn't respond to ctrl+     

I tried some other GTK themes  
But
Synaptic has a title bar that is following the plasma theme  
I'm good this way  
![Ssho2](https://github.com/user-attachments/assets/fafc5902-4ed8-4fae-bcad-abb96f995403)
  
Spiral kde install is so good
